app/src/main/kotlin/io/homeassistant/companion/android/util/InsetsUtil.kt

```diff
@@ -0,0 +1,59 @@
+package io.homeassistant.companion.android.util
+
+import android.view.View
+import androidx.compose.foundation.layout.PaddingValues
+import androidx.compose.foundation.layout.WindowInsets
+import androidx.compose.foundation.layout.WindowInsetsSides
+import androidx.compose.foundation.layout.asPaddingValues
+import androidx.compose.foundation.layout.only
+import androidx.compose.runtime.Composable
+import androidx.core.view.ViewCompat
+import androidx.core.view.WindowInsetsCompat
+import androidx.core.view.updatePadding
+import androidx.preference.PreferenceFragmentCompat
+
+@Composable
+fun WindowInsets.bottomPaddingValues(): PaddingValues {
+    return only(WindowInsetsSides.Bottom).asPaddingValues()
+}
+
+fun PreferenceFragmentCompat.applyBottomSystemBarsInsets(consumeInsets: Boolean = true) {
```

Comment:
Why did you make it possible to set `consumeInsets` to false, when the default value is always used? I can see it being useful if the function is very generic but right now it isn't needed and makes it possible for someone to use it incorrectly in the future. (same applies to `View.applyBottomSystemBarsInsets`)

Reply:
I've made this function for a general use case, I exposed it so that while applying to a screen we know that this can be set mostly. For now I don't know all the places where we will have to apply insets, I propose you that we remove this later if we don't use it? I don't think it's a huge issue here.